marlin/src/ahp/verifier/first_message.rs

```diff
@@ -19,8 +19,6 @@
 pub struct VerifierFirstMessage<F> {
     /// Query for the random polynomial.
     pub alpha: F,
-    /// Randomizer for the lincheck for `A`.
-    pub eta_a: F,
```

Comment:
Instead of computing `eta_a * A + eta_b * B + eta_c * C`, we can just do `A + eta_b * B + eta_c * C`; this is still linearly independent.